<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HTJJPCU" failures="1" tests="6" name="Test" time="11.427" errors="0" timestamp="2024-10-11T15:30:55 IST">
  <testcase classname="api.tests.TestUser" name="testPostUser" time="4.381"/>
  <testcase classname="api.tests.TestUser" name="testGetUser" time="1.241"/>
  <testcase classname="api.tests.TestUser" name="testUpdateUser" time="1.217"/>
  <testcase classname="api.tests.TestUser" name="testGetUser2" time="2.01"/>
  <testcase classname="api.tests.TestUser" name="testDeleteUser" time="1.173"/>
  <testcase classname="api.tests.TestUser" name="testGetUser3" time="1.235">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.TestUser.testGetUser3(TestUser.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testGetUser3 -->
</testsuite> <!-- Test -->
